thin_pieces/barette.scad

```diff
@@ -0,0 +1 @@
+cube(center = true,size = [50,0.5,0.2] );
```

Comment:
What's the purpose of this test? It seems that it's the same test as 1mm_wall.scad.